docs/getting-started-guides/ubuntu/file/etcd/etcd.conf

```diff
@@ -0,0 +1,10 @@
+ETCD_NAME=master
+ETCD_DATA_DIR=/var/lib/etcd
+ETCD_INITIAL_ADVERTISE_PEER_URLS=http://192.168.16.100:2380
+ETCD_INITIAL_CLUSTER=master=http://192.168.16.100:2380
```

Comment:
could you use hostname to replace the specific IP and add a host file to map hostnames and specific IPs
it may confuse elementary user.
e.g. 192.168.16.100 -> master.example.com